release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+[[release-notes-ocp-3-7-release-notes]]
+= {product-title} 3.7 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} is a Platform as a Service (PaaS) that provides
+developers and IT organizations with a cloud application platform for deploying
+new applications on secure, scalable resources with minimal configuration and
+management overhead. {product-title} supports a wide selection of
+programming languages and frameworks, such as Java, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Google Kubernetes, {product-title}
+provides a secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while providing integrated application runtimes
+and libraries. {product-title} brings the OpenShift PaaS platform to customer
+data centers, enabling organizations to implement a private PaaS that meets
+security, privacy, compliance, and governance requirements.
+
+[[ocp-36-about-this-release]]
+== About This Release
+
+Red Hat {product-title} version 3.7
+(link:https://access.redhat.com/errata/RHBA-2017:3188[RHSA-2017:3188]) is now
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v3.7.0-rc.0[OpenShift
+Origin 3.7]. New features, changes, bug fixes, and known issues that pertain to
+{product-title} 3.7 are included in this topic.
+
+{product-title} 3.7 is supported on RHEL 7.3, 7.4, and Atomic Host 7.4.2 and
+newer with the latest packages from Extras, including Docker 1.12.
```

Comment:
Does this need to say 7.4.2 (as I understand it we have a hard requirement on a package only be supported with what will be the latest 7.4.2/3 packages.